docs/guides/insar_product_guide.md

```diff
@@ -0,0 +1,260 @@
+
+InSAR exploits the phase difference between two SAR images to create an interferogram that shows where the phase, and, therefore the distance to the target, has changed from one pass to the next as illustrated in Figure 1.  There are several factors that influence the interferogram including earth curvature, topographic effects, atmospheric delays, surface motion, and noise.  With the proper processing, InSAR can be used to create topographic maps and to detect millimeter scale changes in the earth's surface. Applications include volcanic deformation, subsidence, landslide detection, and earthquake assessment.
+
+### Polarizations
+Polarization refers to the direction of travel of an electromagnetic wave.  A horizontal wave is transmitted so that is oscillates in a plane parallel to the surface imaged, while a vertical wave oscillates in a plane perpendicular to the surface imaged. 
+
+Most modern SAR systems can transmit chirps with either a horizontal or vertical polarization.  In addition, some of these sensors can listen for either horizontal of vertical backscatter.  This gives rise to 4 different types of returns HH, HV, VV, and VH. Here the first letter is the transmit method and the second is the receive method, e.g. VH is a vertically polarized transmit signal with horizontally polarized echoes recorded. 
+
+For InSAR applications, processing is generally performed on the co-pol (VV or HH) data and not on the cross-pol (VH or HV) data. Also, each image used in an InSAR pair is required to be the same polarization - two HH images of the same area could form a valid pair, while a single HH with a single VV of the same area would not.
+
+### Baselines
+
+![Figure 2](../images/baseline.png "Geometry of InSAR Baselines.")
+
+*Figure 2: Geometry of InSAR baselines. Two satellite passes image the same area on the ground from positions S1 and S2, resulting a baseline of B which can be decomposed into normal (B<sub>n</sub>) and perpendicular (B<sub>p</sub>) components.  Here Y is the direction of travel or *along-track* and X is the direction perpendicular to motion, referred to as the *cross-track* or *range* direction.   Credit: Franz J. Meyer*
+
+In order to determine topography, two slightly different vantage points are required.  The term *baseline* refers to the physical distance between these two vantage points. The baseline is decomposed into a normal and a perpendicular components as shown in Figure 2.  Sensitivity to topography depends on the perpendicular baseline, the sensor wavelength, the distance between the satellite and the ground, and the sensor look angle as given in equation 1.  The parameters are diagrammed in Figure 3. 
```

Comment:
I would like to propose that we remove Equation 1 and Figure 3 and their references. This is pretty in-the-weeds about a product that we're not producing, and a use-case that Sentinel-1 isn't even particularly good at. If there are going to be any equations or diagrams of this amount of detail, I would prefer they be about deformation mapping.